.github/workflows/build.yml

```diff
@@ -0,0 +1,43 @@
+on: push
+
+env:
+  SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do you know which actions need this token? I generally prefer keeping the scopes minimal.

Reply:
I agree that usually the scope should be as minimal as possible, but in the case of e.g. `deploy.yml` it used in multiple places (check out [this commit](https://github.com/beatngu13/pdf-zoom-wizard/commit/ada68b5ee4f5afbb8d8b89dd1a5ca24ba54999c1) on a private repo of mine).

My assumption was that as soon as another step needs an environment variable, people tend to simple copy it and then we end up with same thing multiple times. Therefore, I would go for convenience.

What do you think?

Reply:
~Generally, I agree to reduce redundancy. However, actions may choose to use a different variable name for the token as an example~ I was confusing `env`with `with`

So as long as this is no security concern (I mean, we are using official actions right now), go for it.